src/styles/calendar.scss

```diff
@@ -0,0 +1,56 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  position: relative;
+
+  max-width: 707px;
+  margin: 40px auto;
+  padding: 10px;
+
+  &__block {
+    display: block;
+    width: $block-size;
+    height: $block-size;
+    line-height: $block-size;
+    box-sizing: border-box;
+
+    background-color: #eee;
+    border: 1px black solid;
+    text-align: center;
+
+    cursor: pointer;
+    transition-duration: 0.5s;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i}):before {
+        content: "#{$i}";
+        font-family: Arial, Helvetica, sans-serif;
```

Comment:
[checklist #7](https://github.com/mate-academy/layout_calendar/blob/master/checklist.md), don't put styles in for that are identical for all elements (font-size).

Also, font-family belongs in the body, you have one font for the entire page